x/cronos/client/cli/tx.go

```diff
@@ -170,27 +180,35 @@ $ %s tx gov submit-legacy-proposal token-mapping-change gravity0x0000...0000 0x0
 				}
 			}
 
+			signer, err := cmd.Flags().GetString(FlagSigner)
```

Comment:
@yihuang any better approach than need pass governance module account as signer, since it's needed in [message](https://github.com/cosmos/cosmos-sdk/blob/v0.46.2/x/gov/keeper/proposal.go#L39)

Reply:
Maybe we don't need to put signer in proto message and cmd flag, just return the module address in `GetSigner` interface?

Reply:
seems we can't get module address to overwrite in [msgs](https://github.com/cosmos/cosmos-sdk/blob/v0.46.2/x/gov/types/v1/msgs.go)

Reply:
I think we can have the signer field in message, but set to gov module address when build the tx?

Reply:
the [check](https://github.com/cosmos/cosmos-sdk/blob/v0.46.2/x/gov/keeper/proposal.go#L39) says so

Reply:
maybe even `CmdUpdateTokenMapping` but I dare not change without confirm

Reply:
we can deprecate first, without removing directly.